pkgs/utils/src/log.zig

```diff
@@ -347,3 +461,24 @@ fn getLevelColor(comptime level: std.log.Level) []const u8 {
         .debug => Colors.debug,
     };
 }
+
+fn getModuleTagName(tag: ModuleTag) []const u8 {
+    return switch (tag) {
+        .cli => "cli",
+        .chain => "chain",
+        .configs => "configs",
+        .forkchoice => "forkchoice",
+        .gossip_handler => "gossip_handler",
```

Comment:
just gossip is good enough
```suggestion
        .gossip_handler => "gossip",
```